back/src/Guide/Api/Migration.hs

```diff
@@ -0,0 +1,131 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Guide.Api.Migration where
+
+import Imports
+
+import Hasql.Session (Session)
+-- import Hasql.Statement (Statement(..))
+import Hasql.Connection (Connection, Settings)
+
+import qualified Hasql.Session as HS
+-- import qualified Hasql.Decoders as HD
+-- import qualified Hasql.Encoders as HE
+import qualified Data.ByteString as B
+import qualified Hasql.Connection as HC
+
+
+main :: IO ()
+main = do
+  conn <- connection
+  dbCreate conn
+
+connection :: IO Connection
+connection = do
+  eiConnection <- HC.acquire connectionSettings
+  let conn = either (error . show) id eiConnection
+  pure conn
+
+connectionSettings :: Settings
+connectionSettings = HC.settings "localhost" 5432 "postgres" "3" "guide"
+
+dbCreate :: Connection -> IO ()
+dbCreate conn = do
+  result <- mapM (\s -> HS.run s conn)
+    [createTypeProcons, createDbCategories, createDbItems, createDbTraits, createDbUsers, createDbEdits]
+  mapM_ (either print pure) result
```

Comment:
Errors should be printed to stderr, not stdout. In addition, we should crash if any errors happened. So, you should collect all errors with `lefts` and crash if the list is non-empty.